src/Raven.Client/Documents/Operations/AI/Agents/AiAgentConfiguration.cs

```diff
@@ -59,6 +57,12 @@ public AiAgentConfiguration(string connectionStringName, string systemPrompt)
     /// }
     /// </code>
     /// </summary>
+    public string SampleObject { get; set; }
```

Comment:
also the tools has to have the sample object / schema change

Reply:
Done